src/latex.c

```diff
@@ -389,7 +389,6 @@ static int S_render_node(cmark_renderer *renderer, cmark_node *node,
       switch (get_link_type(node)) {
       case URL_AUTOLINK:
         LIT("\\url{");
-        OUT(url, false, URL);
```

Comment:
This fixes the double rendering, but we lose proper escaping for the URL.  (That is what the URL in OUT accomplishes.)  The correct fix would be to keep this line and somehow suppress the output of children of this node.  To do this, return 0 (that stops processing of the contents of the node):
```
-        break;
+       LIT("}");
+       return 0; // don't process contents or we get double render
```